frontend/src/features/admin-form/create/workflow/components/WorkflowContent/EditStepBlock/RespondentBlock.tsx

```diff
@@ -688,12 +694,16 @@ export const RespondentBlock = ({
                 formMethods={formMethods}
                 isLoading={isLoading}
               />
-              <ConditionalRoutingOption
-                selectedWorkflowType={selectedWorkflowType}
-                conditionalFormFields={dropdownFormFields}
-                formMethods={formMethods}
-                isLoading={isLoading}
-              />
+              {/* TODO (MRF-Conditional-Routing): Remove isTest and user check when
```

Comment:
This is the only place the flag needs to be to control the release of the feat to admins.